hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/ipc/MetricsHBaseServerWrapper.java

```diff
@@ -64,4 +65,6 @@ public interface MetricsHBaseServerWrapper {
   int getActiveScanRpcHandlerCount();
 
   long getNettyDmUsage();
+
+  Pair<Long, Long> getTotalAndMaxNettyOutboundBytes();
```

Comment:
Why not introducing two method for getting these two metrics? Just asking...

Reply:
Probably overkill, but was just trying to minimize the chance for these numbers not aligning. Since it's not synchronized, between two loops the totalPendinOutboundBytes can easily change.

Reply:
Skimmed the code again, they are calculated together with a loop, so it is reasonable to return them at once.